cpp/src/arrow/type.h

```diff
@@ -406,6 +407,7 @@ class ARROW_EXPORT FixedSizeBinaryType : public FixedWidthType {
 
   Status Accept(TypeVisitor* visitor) const override;
   std::string ToString() const override;
+  static std::string name() {return "FixedSizeBinary"; }
```

Comment:
can you use `fixed_size_binary` here (that's what's used in the `ToString` implementation)?